Docs - How to cut costs (#9295)
PostHog/posthog.com: contents/docs/product-analytics/cutting-costs.mdx

Author: Thanks for you point on this. I prefer "cutting costs" since I feel it fits in with our brand more and is more opinionated
- Fits in with "we're ridicously cheaper than competitors" (somethign that Andy is adding to our blogs)
- Fits in with "We've decided to make less money narrative"

I see your point on not just a one time thing. Overall I still prefer cutting costs though

---

> title: Cutting costs

Author: ```suggestion
title: Cutting product analytics costs
```

sidebar: Docs
showTitle: true
---

> We aim to be significantly cheaper than our competitors. In addition to our [ridiculously cheap pricing](/pricing), below are tips to reduce your product analytics costs:

Author: Hmm you raise a valid point. Im leaning to either removing the intro completely, or leaving as is. Reasoning:

Someone is probably looking at this page because their PostHog costs are too high. By saying "In addition to our ridiculously cheap pricing", it's probably rubbing salt in their wound.

On the other hand, they're probably wondering if competitors are cheaper if their PostHog costs are high, so it's good to reassure them that they're not. Actually I htink we should leave it in because of this

(sorry for thinking out loud lol)

<HowToCaptureAnonymousEvents />

## Configure autocapture

[Autocapture](/docs/product-analytics/autocapture) is a powerful feature that captures many events automatically. It can also capture more than you need. To reduce which events are captured, you can [set an allow or ignore list](/docs/product-analytics/autocapture#reducing-events-with-an-allow-and-ignorelist).

Alternatively, you can [disable autocapture](/docs/product-analytics/autocapture#disabling-autocapture) completely.

## Only call `identify()` once per session

It's only necessary to [identify](/docs/product-analytics/identify) a user once per session. To prevent sending unnecessary events, check `posthog._isIdentified()` before calling `identify()`:

```js
if (!posthog._isIdentified()) {
    posthog.identify('distinct_id') // Replace 'distinct_id' with your user's unique identifier
}
```

## Only call `group()` once per session

In client-side SDKs, it's only necessary to call [`group()`](/docs/product-analytics/group-analytics) once per session. Prevent calling it multiple times to send fewer events and reduce costs.

## Disable pageview or pageleave events

> PostHog automatically captures pageviews and pageleaves. This is great for product analytics, but it may mean sending more events than you need. An alternative is to disable automatically capturing these events and instead capture them manually for the pages you need.

Reviewer: Simplifying as you mention automatically capture many times
```suggestion
PostHog automatically captures pageviews and pageleaves. This is great for web analytics, but it may capture more events than you need. An alternative is disabling these events and capturing them manually for the pages you need instead.
```

To disable automatically capturing these events, set `capture_pageviews` and `capture_pageleave` to `false` in the configuration options when initializing PostHog:

```js
  posthog.init('<ph_project_api_key>',
    {
      api_host: '<ph_client_api_host>',
      capture_pageviews: false,
      capture_pageleave: false,
    }
  )
```

To manually capture these events, call `posthog.capture('$pageview')` and `posthog.capture('$pageleave')`.

> Note that disabling these events may prevent other PostHog features from working, like [bounce rate](/docs/data/events#how-events-power-posthog).

Reviewer: ```suggestion
> **Note:** Disabling pageview and pageleave events may prevent other PostHog features from working, like [bounce rate](/docs/data/events#how-events-power-posthog).
```
